conf/app.routes

```diff
@@ -31,32 +31,42 @@ GET        /value-weight-of-goods        uk.gov.hmrc.merchandiseinbaggagefronten
 POST       /value-weight-of-goods        uk.gov.hmrc.merchandiseinbaggagefrontend.controllers.ValueWeightOfGoodsController.onSubmit
 
 # SearchGoods
-GET        /search-goods/:idx            uk.gov.hmrc.merchandiseinbaggagefrontend.controllers.SearchGoodsController.onPageLoad(idx: Int)
-POST       /search-goods/:idx            uk.gov.hmrc.merchandiseinbaggagefrontend.controllers.SearchGoodsController.onSubmit(idx: Int)
+GET        /search-goods/:idx            uk.gov.hmrc.merchandiseinbaggagefrontend.controllers.SearchGoodsController.onPageLoad(idx: Int, change: Boolean = false)
```

Comment:
We wouldn't need the routes duplication if we use the state of the GoodsEntry to determine onward navigation.